server/server_test.go

```diff
@@ -295,6 +297,35 @@ func makeOAuth2Tests(clientID string, clientSecret string, now func() time.Time)
 					return nil
 				},
 			},
+			{
+				name: "verify OIDC group prefixing",
```

Comment:
This test ensures that the `mock` prefix is added. (Since that is the connector used in these tests)